### PR TITLE
Drop datablocks from radial average

### PR DESCRIPTION
**Pull request by phyy-nx**

### CODE REVIEW

File: src/dxtbx/command_line/radial_average.py [hunk 1]
```diff
@@ -351,7 +344,7 @@ def load_func(x):
                     results = results.select(xvals <= params.plot_x_max)
                     xvals = xvals.select(xvals <= params.plot_x_max)
                 if params.x_axis == "resolution":
-                    xvals = 1 / (xvals ** 2)
+                    xvals = 1 / (xvals**2)
```
Comment (Anthchirp) [inline src/dxtbx/command_line/radial_average.py:347]: Did you run black manually rather than using pre-commits?

Comment (phyy-nx) [reply src/dxtbx/command_line/radial_average.py:347]: Yah that's what I usually do.  I often don't have precommits available so I just run black and flake8 manually.

Comment (Anthchirp) [reply src/dxtbx/command_line/radial_average.py:347]: That explains that. You're running the wrong version.

Comment (phyy-nx) [reply src/dxtbx/command_line/radial_average.py:347]: I'm using the latest from conda-forge, 22.1.0.  What should I be using?

Comment (phyy-nx) [reply src/dxtbx/command_line/radial_average.py]: Looks like the [pinned version of black](https://github.com/dials/dials/blob/main/.pre-commit-config.yaml#L11) in the precommittbx isn't available in conda-forge for manual installs.  21.5b2 seems to work tho.

**Pull request merged**